### ISSUE TITLE
Register now button not visible clearly

### ISSUE THREAD

**Issue by nown1ne** [NITRR-Open-Source-Community/NOSC-Website#106]
**Description**
The register now button has limited visibility due to the usage of same text and background color

**Difficulty**
Good first issue

**To Reproduce**
1. Go to 'nosc.netlify.app'
2. Notice the register now button

**Expected behavior**
The button text should be clearly visible

**Screenshots**
![image](https://github.com/NITRR-Open-Source-Community/NOSC-Website/assets/25835195/dd2229dc-2844-4764-8b79-dee0379e5740)

![image](https://github.com/NITRR-Open-Source-Community/NOSC-Website/assets/25835195/87c21e44-ad28-41b8-ab7f-8d1461dad539)

**Comment by Neerajmeka25** [comment]
Hey @nown1ne,
Can you assign me the work, I would like to work on it.

**Comment by princyballabh** [comment]
Hey, I have knowledge about html and css and would like to fix this issue.

**Comment by priyanshu-gupta0503** [comment]
hey@nown1ne  I have already successfully completed two of my pr's so I have a little but good enough knowledge of it kindly assign this task to me..........

**Comment by DaSeeker67** [comment]
hello @princyballabh  And @Neerajmeka25  we work on a first come first serve basis so we can assign you this task but star this repo so that you can get notifications of upcoming issues and you can be assigned first

**Comment by priyanshu-gupta0503** [comment]
@DaSeeker67  assign this to me I have star the repo

**Comment by princyballabh** [comment]
@DaSeeker67 I have starred the repo

**Comment by techemmy** [comment]
@nown1ne, I made a PR to fix this issue 2 days ago, but you haven't reviewed it since.
